templates/todo/web/react-fluent/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "todo-vite-note-mongo-aca",
+  "name": "todo-vite-node-mongo-aca",
```

Comment:
should it be `todo-vite-node-mongo-aks` ?

Reply:
This was a minor change to fix typo of `note` to `node`.  I've gone ahead and removed the service target suffix from this app since it's not important and we don't need to maintain different versions of this file per template.